src/Connection/InetSocket.php

```diff
@@ -14,36 +14,28 @@ abstract class InetSocket implements Connection
 
     /**
      * host name
-     *
-     * @var string
      */
-    protected $host;
+    protected string $host;
```

Comment:
This represents a breaking change (like all explicit types on protected properies in this class). To avoid this unnecessary BC you might want to revert these changes,